mongoapi/dbtestapp/Dockerfile

```diff
@@ -4,15 +4,15 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-FROM golang:1.18.0-stretch AS test
+FROM golang:1.21.3-bookworm AS test
 
 LABEL maintainer="ONF <omec-dev@opennetworking.org>"
 
 RUN apt-get update && apt-get -y install vim 
 
 RUN cd $GOPATH/src && mkdir -p dbtestapp
 COPY . $GOPATH/src/dbtestapp
-RUN cd $GOPATH/src/dbtestapp && go install 
+RUN cd $GOPATH/src/dbtestapp && go install
```

Comment:
What about simply doing something like this?
```go
WORKDIR $GOPATH/src/dbtestapp
COPY . .
RUN go install
```

Reply:
Done